docs/FeatureManagement/Samples/RequirementType.sample.json

```diff
@@ -0,0 +1,144 @@
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "RequirementTypeAnyFirstFilterPassed",
+                "description": "Same as DefaultRequirementTypeFirstFilterPassed, but requirement type is specified. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        }
+                    ],
+                    "requirement_type": "Any"
+                }
+            },
+            {
+                "id": "RequirementTypeAnyLastFilterPassed",
+                "description": "Same as DefaultRequirementTypeLastFilterPassed, but requirement type is specified. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ],
+                    "requirement_type": "Any"
+                }
+            },
+            {
+                "id": "RequirementTypeAllPassed",
+                "description": "Requirement type All. Will always return false.",
```

Comment:
For "RequirementTypeAllPassed" test case, it says `Will always return false.` in the description. Why it always return false?
The 1st and 2nd fiters should both return true?